x-pack/plugins/security_solution/public/common/components/event_details/table/summary_table.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import styled from 'styled-components';
+import { EuiInMemoryTable } from '@elastic/eui';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const SummaryTable = styled(EuiInMemoryTable as any)`
```

Comment:
nit: to avoid disabling the linter, consider changing:

```ts
import styled from 'styled-components';
// ...
// eslint-disable-next-line @typescript-eslint/no-explicit-any
export const SummaryTable = styled(EuiInMemoryTable as any)
```

to something like:

```ts
import styled, { AnyStyledComponent } from 'styled-components';
// ...
export const SummaryTable = styled(EuiInMemoryTable as unknown as AnyStyledComponent)`
```

Reply:
Done, thanks!